Presentation.Web/app/components/data-processing/data-processing-registration-overview.controller.ts

```diff
@@ -453,6 +453,18 @@
                             .withInitialVisibility(false)
                             .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.LastChangedAt))
                             .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderDate(dataItem.LastChangedAt)))
+                    .withColumn(builder =>
+                        builder
+                            .withDataSourceName("SystemUuidsAsCsv")
+                            .withTitle("IT Systemer (UUID)")
+                            .withId("itSystemUuid")
+                            .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
+                            .withRendering(
+                                dataItem => Helpers.RenderFieldsHelper.renderReference(dataItem.SystemUuidsAsCsv,
+                                    dataItem.SystemUuidsAsCsv))
+                            .withExcelOutput(
+                                dataItem => Helpers.ExcelExportHelper.renderReference(dataItem.SystemUuidsAsCsv,
```

Comment:
no need for additional excel specific rendering if  the strategy is the same as the main rendering

Reply:
wrong render method used also (renderReference ???)